smoke_tests_fpm.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -e
+
+# TODO: we should need only one web/app.php file?
+# use $ENVIRONMENT_NAME with a default?
```

Comment:
ELPP-3367 might well head that way.

Reply:
I think it can be done right now in this service, because the only difference between is a string containing `$ENVIRONMENT_NAME`